.env.example

```diff
@@ -19,44 +17,31 @@ CHRONOS_SCHEDULE_JOBS='0 */10 * * * *'
 CHRONOS_UPDATE_TOKENS='0 0 0 1,15 * *'
 CHRONOS_PROCESS_RECURRENCE='0 */5 * * * *'
 CHRONOS_AUTOPAUSE='0 0 5 * * *'
-CI='true'
+# Enter a short url redirect service for invitations, it needs to redirecto to /invitation-link
+# INVITATION_SHORTLINK='example.com'
+
+# AUTH
+AUTH_INTERNAL_DISABLED='false'
+AUTH_GOOGLE_DISABLED='false'
+AUTH_SSO_DISABLED='false'
+OAUTH2_REDIRECT=''
+
+# ANALYTICS
+# AMPLITUDE_WRITE_KEY='key_AMPLITUDE_WRITE_KEY'
 DD_AGENT_HOST='172.17.0.1'
 DD_API_KEY='DD_API_KEY'
 DD_APM_ENABLED='false'
 DD_TRACE_ENABLED='false'
 DD_ENV='local'
-FILE_STORE_PROVIDER='s3'
-GITHUB_CLIENT_ID='key_GITHUB_CLIENT_ID'
-GITHUB_CLIENT_SECRET='key_GITHUB_CLIENT_SECRET'
-GITHUB_WEBHOOK_SECRET='key_GITHUB_WEBHOOK_SECRET'
-GITLAB_CLIENT_ID='key_GITLAB_CLIENT_ID'
-GITLAB_CLIENT_SECRET='key_GITLAB_CLIENT_SECRET'
-GOOGLE_CLOUD_CLIENT_EMAIL='key_GOOGLE_CLOUD_CLIENT_EMAIL'
-GOOGLE_CLOUD_PRIVATE_KEY='key_GOOGLE_CLOUD_PRIVATE_KEY'
-GOOGLE_CLOUD_PRIVATE_KEY_ID='key_GOOGLE_CLOUD_PRIVATE_KEY_ID'
+# SEGMENT_WRITE_KEY='key_SEGMENT_WRITE_KEY'
+SENTRY_AUTH_TOKEN='key_SENTRY_AUTH_TOKEN'
+# SENTRY_DSN='key_SENTRY_DSN'
+
+# DB RELATED
 GRAPHQL_HOST='localhost:3000'
 GRAPHQL_PROTOCOL='http'
```

Comment:
-1 this belongs in the `# important` section, together with `HOST`, `PORT` and `PROTO`

Reply:
Done ✅